slurp/src/main/webapp/scripts/utils.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Parse URL <pre><code>NAME.html?key1=value1&key2=value2...</code></pre> into {key1: value1, ...}.
+ * @param {string} url The URL to be parsed. 
+ * @returns {Object.<string, string>} Dictionary of the parsed URL parameters. 
+ */
+function parseUrl(url) {
```

Comment:
I think this functionality is taken care of using [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)?

Reply:
yup, and it's more versatile. I'll leave it in here and replace it with URLSearchParams in a later PR (#28)

Reply:
Coolio. 👍